src/main/java/io/quarkus/gizmo2/impl/InterfaceCreatorImpl.java

```diff
@@ -48,6 +52,8 @@ public MethodDesc method(final String name, final Consumer<AbstractMethodCreator
     }
 
     void accept(final Consumer<InterfaceCreator> builder) {
+        preAccept();
```

Comment:
`preAccept()` and `postAccept()` must be called in order to propagate flags etc.